Framework/PythonInterface/core/inc/MantidPythonInterface/core/PythonStdoutChannel.h

```diff
@@ -17,8 +17,17 @@
 
 #pragma once
 
+// local includes
 #include "MantidPythonInterface/core/DllConfig.h"
+#include "MantidPythonInterface/core/WrapPython.h"
+
+// 3rd-party includes
+#include "MantidKernel/StdoutChannel.h"
 #include <Poco/ConsoleChannel.h>
+//#include <pybind11/iostream.h>
+
+// standard includes
+#include <iostream>
```

Comment:
Can we remove this from the header? It's very heavy to include and we shouldn't need to include it in a header.

Reply:
thanks for catching this. It was a leftover from a console channel class I had written before and deleted later. I'll remove these lines